docs/source/requirements.rst

```diff
@@ -126,7 +126,7 @@ Build system:
 =============
 
 * CMake >= 3.16: required
-* CMake >= 3.18: Fortran linkage. This does not affect most mixed Fortran/Kokkos builds. See [build issues](BUILD.md#KnownIssues).
+* CMake >= 3.18: Fortran linkage. This does not affect most mixed Fortran/Kokkos builds. See https://github.com/kokkos/kokkos/blob/master/BUILD.md#known-issues.
```

Comment:
```suggestion
* CMake >= 3.18: Fortran linkage. This does not affect most mixed Fortran/Kokkos builds. See 'known build issues <https://github.com/kokkos/kokkos/blob/master/BUILD.md#known-issues>'_.
```

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Reply:
your suggestion looks like this:

<img width="898" alt="image" src="https://github.com/kokkos/kokkos-core-wiki/assets/18708772/104a6bda-dc99-4235-898a-48fc9a994530">